src/api/plugins/video/warren_video/api.py

```diff
@@ -34,6 +34,7 @@ async def views(
 ) -> DailyCounts:
     """Number of views for `video_id` in the `since` -> `until` date range."""
     # Switch/case pattern matching with the (complete, unique) boolean tuple
+    logger.info("Start computing 'views' indicator")
```

Comment:
It would add more complexity than necessary, but I could have to reuse the function's name instead of duplicating it. IMO the duplication is fine.

Reply:
This is not something we want to log at this level. The access logs are already here for this.

```suggestion
    logger.debug("Start computing 'views' indicator")
```

Reply:
Alright, gotcha. In a production setup, would the Uvicorn access logs and the logger's ones be on the same stream?

Reply:
In a kubernetes context, your application should log everything in standard unix streams (stdin, stdout), or dedicated logging services/tools. Apart from that, IMO this message is not relevant for a production environment: what would you expect from it?

Reply:
Afterthoughts, I do agree with you.